src/unfold/templates/unfold/helpers/userlinks.html

```diff
@@ -7,13 +7,19 @@
 
         <input type="checkbox" class="hidden peer" id="show-userlinks" />
 
-        <div class="absolute bg-white border flex-col hidden leading-none py-1 right-0 rounded-md shadow-lg text-sm text-gray-500 top-10 w-48 peer-checked:flex">
+        <div class="absolute bg-white border flex-col hidden leading-none py-1 right-0 rounded-md shadow-lg text-sm text-gray-500 top-10 min-w-[12rem] max-w-[18rem] peer-checked:flex">
             {% if nav_global %}
                 {{ nav_global }}
 
                 <div class="bg-gray-100 h-px my-1"></div>
             {% endif %}
 
+            <div class="truncate flex justify-start flex-col items-start mx-4 ">
+                <p class="cursor-pointer text-sm leading-5 text-gray-500">{{ user.first_name }} {{ user.last_name }}</p>
+                <p class="cursor-pointer text-xxs leading-3 text-gray-400">{{ user.email }}</p>
```

Comment:
E-mail is a blank field in DB model as well.

Reply:
without email: 
![image](https://user-images.githubusercontent.com/24371558/202234248-2ee6dad9-a2f8-478f-ad9c-7683b1ea121f.png)

with only email: 
![image](https://user-images.githubusercontent.com/24371558/202234144-76c53eaa-9ecc-4474-b7de-59652b9bef60.png)

full:
![image](https://user-images.githubusercontent.com/24371558/202234809-bac42c37-6895-4509-a892-ab1378fb4d77.png)